### PR TITLE
Dev to main

### PR DESCRIPTION
**Pull request by OscarFMdev**
# PR description: 📄 
In this branch I:
- Created a single-page application using React, Redux, and an API. :atom: 
- Deployed the project using Netlify. 💯 
- Recorded a [video](https://github.com/OscarFMdev/countries-info) explanation of the project. 🎥

## IMPORTANT NOTE
**Dear code reviewer, for some reason the linters are not working remotely, but I send you a picture of the linters passing locally:**
image

#### Thank you for reviewing my code! 🙌🏽

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,46 +1,192 @@
...
+If you like this project show your support giving a ⭐.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank [REST Countries API](https://restcountries.com/) for making this project possible.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```
Comment (omar-muhamad) [inline README.md]: - [x] As per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md#:~:text=The%20Creative%20Commons%20license%20of%20the%20design%20requires%20that%20you%20give%20appropriate%20credit%20to%20the%20author.%20Therefore%2C%20you%20must%20do%20it%20in%20the%20README%20of%20your%20project.), Please give appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/). At the `Acknowledgments` section.

File: src/pages/Countries.module.css [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  height: 22vh;
+}
+
+.countryContainer:hover,
+.countryContainer:active {
+  background-color: #4369b2;
+}
+
+.countryName {
+  font-size: 20px;
```
Comment (omar-muhamad) [inline src/pages/Countries.module.css]: - [x] As per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md#:~:text=Layout%3A%20composition%20and%20space%20between%20elements.), Please make the list of countries appear as two columns and the background color is staggered like a chess board to match the given design.

| Given Design  | Your Design |
| ------------- | ------------- |
| image  | image |

File: src/App.module.css [hunk 3]
```diff
@@ -2,8 +2,14 @@
   margin: 0;
   padding: 0;
   box-sizing: border-box;
+  font-family: 'Montserrat', sans-serif;
 }
```
Comment (omar-muhamad) [inline src/App.module.css:6]: - [ ] As per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md#:~:text=Typography%3A%20font%20face%2C%20size%20and%20weight.), Please add the correct font families for your app,` Gill sans` and `Lato` fonts, not `Montserrat` as you did.

File: src/pages/CountryInfo.module.css [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+}
+
+.flag {
+  border: 1px solid black;
+}
+
+.infoList {
+  list-style: none;
+  display: flex;
+  flex-direction: column;
```
Comment (omar-muhamad) [inline src/pages/CountryInfo.module.css:23]: - [ ] Nice job adding the back button arrow to the details page, but Kindly add it at the left of the nav bar to match the given design.

- [ ] The country details elements are not centered and overflowed from the right, Kindly fix that.

| Given Design  | Your Design |
| ------------- | ------------- |
| image  | image  |

**Review by omar-muhamad — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @OscarFMdev,

Nice job, you are almost there.



## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- You used the correct workflow.  ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [ ] Please check the inline comments below.

## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by omar-muhamad** [comment]
Hi @OscarFMdev,
As you mentioned the linters are working locally.

**Comment by OscarFMdev** [comment]
@Omar-Muhamad Thank you very much for the full review, and most importantly for helping me improve my design and README file. Have a great day!

**Review by emmanuellekamwa — APPROVED** [review]
**STATUS: APPROVED 🍾✔🎉**
Hello @OscarFMdev 👋
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- You documented your work professionally.✔
- You asked for the changes in a polite and friendly way.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by emmanuellekamwa — APPROVED** [review]
**STATUS: APPROVED 🍾✔🎉**
Hello @OscarFMdev 👋
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- You documented your work professionally.✔
- You asked for the changes in a polite and friendly way.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**